packages/fetchai/connections/p2p_libp2p/aea/api.go

```diff
@@ -318,9 +317,11 @@ func read_envelope(pipe *os.File) (*Envelope, error) {
 /*
 
   Sandbox
+  - DISABLED
```

Comment:
Why has the sandbox been disabled?

Reply:
Sandbox is disabled to bypass the issue of implementing `syscall.Mkfifo` for Windows platform.
Although Microsoft provides an official implementation for it, the Sandbox is not a priority as of now.